Homework/homework5/task2.py

```diff
@@ -0,0 +1,39 @@
+"""
+Написать декоратор.
+
+Который позволит сохранять информацию из
+исходной функции (__name__ and __doc__), а так же сохранит саму
+исходную функцию в атрибуте __original_func
+print_result изменять нельзя, за исключением добавления вашего
+декоратора на строку отведенную под него - замените комментарий
+До применения вашего декоратор будет вызываться AttributeError при custom_sum.__original_func
+Это корректное поведение
+После применения там должна быть исходная функция
+Ожидаемый результат:
+print(custom_sum.__doc__)  # 'This function can sum any objects which have __add___'
+print(custom_sum.__name__)  # 'custom_sum'
+print(custom_sum.__original_func)  # <function custom_sum at <some_id>>
+"""
+
+from typing import Callable
+
+
+def wraps(wrapped: Callable) -> Callable:
```

Comment:
can you find an analog in the standard library?

Reply:
Yes, https://docs.python.org/3/library/functools.html#functools.wraps